src/js/custom-javascript.js

```diff
@@ -1 +1,36 @@
+/**
+ * --------------------------------------------------------------------------
+ * Understrap (v1.0.0): index.js
+ * Licensed under GPL (http://www.gnu.org/licenses/old-licenses/gpl-2.0.en.html)
+ * --------------------------------------------------------------------------
+ */
+
+import Alert from '../../node_modules/bootstrap/js/src/alert'
```

Comment:
use `https://github.com/rollup/plugins/tree/master/packages/node-resolve` so that you don't need the `.../.../node_modules/` part

Reply:
👍